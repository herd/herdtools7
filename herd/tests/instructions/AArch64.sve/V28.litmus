AArch64 V28
{}
P0               ;
MOV Z0.S,#1      ;
PTRUE P0.S,VL2   ;
SETFFR           ;
RDFFR P1.B,P0/Z  ;
UADDV D2,P1,Z0.S ;

forall 0:V2.4S={2,0,0,0}

