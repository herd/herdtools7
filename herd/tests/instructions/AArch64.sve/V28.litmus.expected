Test V28 Required
States 1
0:V2.4S={2,0,0,0};
Ok
Witnesses
Positive: 1 Negative: 0
Flag Scalable-Vector-Extensions-is-work-in-progress
Condition forall (0:V2.4S={2,0,0,0})
Observation V28 Always 1 0
Hash=b9acf152c7bad27dd72291dbcd60e586

